### DISCUSSION TITLE
How can I preview responsive ChartJS within Iframe on my cefsharp browser?

### DISCUSSION THREAD

**Discussion by J5LAM** [cefsharp/CefSharp#4398] [General]
I'm having issues on presenting some responsive  graphs based on chartjs within an iframe element on my cefsharp web browser where I pretty much get the google chromes equivalent of an aw snap issue. 

Has anyone used cefsharp to show the webrowser with a chartjs "backend"?

**Comment by amaitland** [comment]
> pretty much get the google chromes equivalent of an aw snap issue.

What is the error you are seeing? What version are you using?
